layouts/partials/left_content.html

```diff
@@ -0,0 +1,35 @@
+<div class="row">
+  <div class="col s4 offset-s4 m12">
+    <img class="logo hoverable responsive-img" src="/img/logo.png" alt="NottsJS Logo" />
```

Comment:
The images should be relative. I've noticed a few broken on your pages site https://hawx.github.io/nottsjs.github.io/